pkg/tarmak/assets/assets.go

```diff
@@ -8,4 +8,5 @@
 
 package assets
 
-//go:generate go-bindata -prefix ../../../ -pkg $GOPACKAGE -o assets_bindata.go ../../../terraform/amazon/modules/... ../../../terraform/amazon/templates/... ../../../puppet/... ../../../packer/...
+//go:generate go run ../../../cmd/tagging_control/main.go zip ../../../tagging_control.zip ../../../tagging_control_linux_amd64
```

Comment:
This is really ugly, not too sure if it break during the release process

Reply:
Why would this break?